<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1218</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,9">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_side" native="true">
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="2,0">
           <property name="spacing">
            <number>1</number>
           </property>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,0,8">
             <item>
              <spacer name="verticalSpacer_3">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <widget class="QPushButton" name="btn_home_dataset">
                 <property name="text">
                  <string>数据处理</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btn_home_img_process">
                 <property name="text">
                  <string>图像处理</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btn_home_count">
                 <property name="text">
                  <string>计数统计</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btn_home_yolo">
                 <property name="text">
                  <string>yolo检测</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btn_home_other">
                 <property name="text">
                  <string>其他</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <spacer name="verticalSpacer_4">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="btn_side">
               <property name="maximumSize">
                <size>
                 <width>10</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="mres.qrc">
                 <normaloff>:/images/ico/ico_left.png</normaloff>:/images/ico/ico_left.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="currentIndex">
         <number>2</number>
        </property>
        <widget class="QWidget" name="page_home">
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>291</x>
            <y>40</y>
            <width>111</width>
            <height>21</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Alef</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Home</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="page_img">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QTabWidget" name="tabWidget">
            <property name="tabPosition">
             <enum>QTabWidget::West</enum>
            </property>
            <property name="tabShape">
             <enum>QTabWidget::Triangular</enum>
            </property>
            <property name="currentIndex">
             <number>1</number>
            </property>
            <widget class="QWidget" name="tab_3">
             <attribute name="title">
              <string>页</string>
             </attribute>
            </widget>
            <widget class="QWidget" name="tab">
             <attribute name="title">
              <string>阈值分割</string>
             </attribute>
             <layout class="QGridLayout" name="gridLayout_4">
              <item row="0" column="0">
               <layout class="QVBoxLayout" name="verticalLayout" stretch="8,0">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout">
                  <item>
                   <widget class="QLabel" name="label_v_src">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="minimumSize">
                     <size>
                      <width>360</width>
                      <height>240</height>
                     </size>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">border: 1px solid black;</string>
                    </property>
                    <property name="text">
                     <string>原始图像</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="label_v_dst">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="minimumSize">
                     <size>
                      <width>360</width>
                      <height>240</height>
                     </size>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">border:2px dotted #242424;</string>
                    </property>
                    <property name="text">
                     <string>结果</string>
                    </property>
                    <property name="textFormat">
                     <enum>Qt::AutoText</enum>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_6">
                  <item>
                   <spacer name="horizontalSpacer_4">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeType">
                     <enum>QSizePolicy::Preferred</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_2">
                    <property name="sizeConstraint">
                     <enum>QLayout::SetMinimumSize</enum>
                    </property>
                    <item>
                     <widget class="QPushButton" name="btn_open_img">
                      <property name="maximumSize">
                       <size>
                        <width>80</width>
                        <height>16777215</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>选择图像</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QSlider" name="horizontalSlider">
                      <property name="maximum">
                       <number>255</number>
                      </property>
                      <property name="value">
                       <number>20</number>
                      </property>
                      <property name="orientation">
                       <enum>Qt::Horizontal</enum>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QSpinBox" name="spinBox">
                      <property name="maximum">
                       <number>255</number>
                      </property>
                      <property name="value">
                       <number>20</number>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="btn_v_clear">
                      <property name="maximumSize">
                       <size>
                        <width>80</width>
                        <height>16777215</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>clear</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer_6">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeType">
                     <enum>QSizePolicy::Preferred</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="tab_2">
             <attribute name="title">
              <string>待定</string>
             </attribute>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_yolo">
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QTabWidget" name="tabWidget_2">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="tab_yolo_detect">
             <attribute name="title">
              <string>图像检测</string>
             </attribute>
             <layout class="QGridLayout" name="gridLayout_6">
              <item row="0" column="0">
               <widget class="QSplitter" name="splitter">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <widget class="QWidget" name="layoutWidget">
                 <layout class="QHBoxLayout" name="horizontalLayout_7">
                  <item>
                   <spacer name="horizontalSpacer_5">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeType">
                     <enum>QSizePolicy::Minimum</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_5">
                    <property name="sizeConstraint">
                     <enum>QLayout::SetMinimumSize</enum>
                    </property>
                    <item>
                     <widget class="QPushButton" name="btn_yolo_open_file">
                      <property name="text">
                       <string>选择文件</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="lineEdit_yolo_file_path"/>
                    </item>
                    <item>
                     <layout class="QHBoxLayout" name="horizontalLayout_11">
                      <property name="sizeConstraint">
                       <enum>QLayout::SetMinimumSize</enum>
                      </property>
                      <item>
                       <widget class="QPushButton" name="btn_yolo_open_weight">
                        <property name="text">
                         <string>选择权重</string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QLineEdit" name="lineEdit_yolo_weight_path"/>
                      </item>
                     </layout>
                    </item>
                    <item>
                     <widget class="QPushButton" name="btn_yolo_clear">
                      <property name="text">
                       <string>clear</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer_7">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeType">
                     <enum>QSizePolicy::Preferred</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </widget>
                <widget class="QWidget" name="layoutWidget_2">
                 <layout class="QHBoxLayout" name="horizontalLayout_8">
                  <item>
                   <widget class="QLabel" name="label_yolo_src">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="minimumSize">
                     <size>
                      <width>360</width>
                      <height>240</height>
                     </size>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">border: 1px solid black;</string>
                    </property>
                    <property name="text">
                     <string>原始文件</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="label_yolo_dst">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="minimumSize">
                     <size>
                      <width>360</width>
                      <height>240</height>
                     </size>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">border:2px dotted #242424;</string>
                    </property>
                    <property name="text">
                     <string>结果</string>
                    </property>
                    <property name="textFormat">
                     <enum>Qt::AutoText</enum>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
                <widget class="QWidget" name="layoutWidget_3">
                 <layout class="QHBoxLayout" name="horizontalLayout_9">
                  <item>
                   <spacer name="horizontalSpacer_3">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeType">
                     <enum>QSizePolicy::Minimum</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_10">
                    <item>
                     <widget class="QPushButton" name="btn_yolo_detect">
                      <property name="text">
                       <string>检测</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="btn_yolo_save">
                      <property name="text">
                       <string>存储</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="btn_yolo_save_file">
                      <property name="text">
                       <string>存储文件</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="btn_yolo_stop">
                      <property name="text">
                       <string>其他</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer_2">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeType">
                     <enum>QSizePolicy::Minimum</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </widget>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="tab_yolo_img">
             <attribute name="title">
              <string>Tab 2</string>
             </attribute>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_3"/>
        <widget class="QWidget" name="page">
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>140</x>
            <y>224</y>
            <width>141</width>
            <height>81</height>
           </rect>
          </property>
          <property name="text">
           <string>数据处理</string>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1218</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="mres.qrc"/>
 </resources>
 <connections/>
</ui>
